roles/satellite-clone/tasks/main.yml

```diff
@@ -120,6 +120,10 @@
 - name: Wait for foreman-tasks service to start
   pause: minutes=5
 
+- name: Run installer upgrade
+  command: satellite-installer --upgrade
+  when: satellite_version == 6.2
```

Comment:
something like

- name: Run installer upgrade (Currently for Migration and 6.2 cloning only)